scripts/package.sh

```diff
@@ -20,7 +20,7 @@ if [ ! -d "$BUILD_DIR" ]; then
 fi
 
 echo "> Building Socorro $BUILD_VERSION ..."
-if [ "$BUILD_TYPE" != "tar" ]; then
+if [ "$BUILD_TYPE" == "rpm" ]; then
```

Comment:
We changed "tar" to "ci". If it's not "ci" then it is "rpm".

This block would build an rpm or a deb depending on the value of `BUILD_TYPE`. We don't do debs anymore, so that lets me convert this `if` test to "is it rpm?".